docs/MODULES.md

```diff
@@ -303,18 +303,56 @@ or with IPv6 and subnet
 
 ##### IPTunnel Client and Internet Peer
 
-If you wish to run an IPTunnel client on the same node as as an Internet peer, you will need to create a route to that node over the Internet connection available.
+If you wish to run an IPTunnel client on the same node as as an Internet peer, you will need to use a method of rerouting the path to the Yggdrasill peers Internet connection available instead over yggdrasil.
```

Comment:
```suggestion
If you wish to run an IPTunnel client on the same node as an Internet peer, you will need to ensure the path to the Yggdrasil peer goes over the Internet connection instead of using the tunnel.
```